pensieve/pensieve.py

```diff
@@ -2,96 +2,181 @@
 import spacy
 import os
 import textacy
-from collections import defaultdict
+import pickle
+from numpy.random import randint
 from collections import Counter
 
+print('Loading spaCy...')
 nlp = spacy.load('en')
 
 
 class Corpus(object):
 
     def __init__(self, corpus_dir=None):
+        """
+        Corpus to extract mems from
+
+        Args:
+            corpus_dir: Directory containing all corpus files
+
+        Attributes:
+            docs: List of Doc objects from corpus
+            paragraphs: List of Paragraph objects from corpus
+        """
         self.corpus_dir = corpus_dir
-        self.docs = self.read_corpus(corpus_dir)
+        self.docs = self.read_corpus()
```

Comment:
Good improvement using class attributes

No change needed, but possibly move processing out of init?